src/validate.rs

```diff
@@ -15,7 +15,7 @@ use crate::game::Game;
 use crate::item::Item;
 use crate::lowercase::Lowercase;
 use crate::report::{err, error, fatal, old_warn, report, warn, Confidence, ErrorKey, Severity};
-use crate::scopes::{scope_prefix, scope_to_scope, Scopes};
+use crate::scopes::{scope_prefix, scope_to_scope};
```

Comment:
Why this change? I prefer to import everything at the top so that there are no surprises

Reply:
It is unused import when building vic3-tiger

Reply:
I see :)